src/Microsoft.DocAsCode.HtmlToPdf/ConvertWrapper.cs

```diff
@@ -282,6 +278,21 @@ private void ConvertCore(string basePath, string pdfFileName, IList<HtmlModel> h
 
             converter.Save(Path.Combine(_pdfOptions.DestDirectory, pdfFileName));
         }
+        private bool IsType(string type, ManifestItemType targetType)
+        {
+            if (Enum.TryParse(type, out ManifestItemType actualType))
```

Comment:
>TryParse [](start = 21, length = 8)

ignore case? and if can parse from enum, still need to compare with target type?